YoutubeDLSharp/Utils.cs

```diff
@@ -80,5 +89,201 @@ public static string GetFullPath(string fileName)
             }
             return null;
         }
+
+#region Download Helpers
+
+        public static string YtDlpBinaryName => GetYtDlpBinaryName();
+        public static string FfmpegBinaryName => GetFfmpegBinaryName();
+        public static string FfprobeBinaryName => GetFfprobeBinaryName();
+
+        public static async Task DownloadBinaries(bool skipExisting = true, string directoryPath = "")
+        {
+            if (skipExisting)
+            {
+                if(!File.Exists(Path.Combine(directoryPath, GetYtDlpBinaryName())))
+                {
+                    await DownloadYtDlp(directoryPath);
+                }
+                if (!File.Exists(Path.Combine(directoryPath, GetFfmpegBinaryName())))
+                {
+                    await DownloadFFmpeg(directoryPath);
+                }
+                if (!File.Exists(Path.Combine(directoryPath, GetFfprobeBinaryName())))
+                {
+                    await DownloadFFprobe(directoryPath);
+                }
+            }
+            else
+            {
+                await DownloadYtDlp(directoryPath);
+                await DownloadFFmpeg(directoryPath);
+                await DownloadFFprobe(directoryPath);
+            }            
+        }
+
+        private static string GetYtDlpBinaryName(bool fullPath = false)
```

Comment:
Imho this method is a little confusing as it can either return a download url or a binary name, which are two separate things. Why not have one method for the binary name (same as for the other binaries) and another for the download url?